artifactory/README.md

```diff
@@ -0,0 +1,33 @@
+# Habitat package: artifactory (open source)
+
+Artifactory is an advanced Binary Repository Manager for use by build tools (like Maven and Gradle), dependency management tools (like Ivy and NuGet) and build servers (like Jenkins, Hudson, TeamCity and Bamboo).
+
+## Maintainers
+
+* [Scott Ford](https://github.com/smford22)
```

Comment:
Not absolutely required, but if you want to be the maintainer for the package, you might want to consider adding yourself to `CODEOWNERS` also. 😄

Reply:
Done. Maintainers section has been removed from the README